### DISCUSSION TITLE
How to use a variable to receive an error report on the eslint console?

### DISCUSSION THREAD

**Discussion by Adrian-Zephyr-Liao** [eslint/eslint#15983] [Q&A]
`spawn.sync(
      "npx",
      [
        "eslint",
        "-c",
        path.resolve(__dirname, `../../eslint/${framework}.js`),
        ...eslint,
        "--fix",
        "--no-inline-config",
        "-f",
        "json",
        "-o",
        path.resolve(process.cwd(), "eslint.report.json")
      ],
      {
        stdio: "inherit",
      }
    );`
I want to get an error message in the JSON format in Node, but it can only be output to the file.

**Comment by sam3k** [comment]
Hi @RainyLiao, I am not sure I understand what you're asking. Can you give some clear step by steps information?
